website/content/docs/nia/configuration.mdx

```diff
@@ -343,9 +348,14 @@ task {
 
 ### Task Source Input
 
-You can add an optional `source_input` block to the `task` block. The `source_input` block specifies a Consul object containing values or metadata to be provided to the Terraform Module. CTS supports the following source input types.
+You can optionally add one or more `source_input` blocks to the `task` block. A `source_input` block specifies a Consul object containing values or metadata to be provided to the Terraform Module. Both scheduled and dynamic tasks can be configured with `source_input` blocks.
+
+There are some limitations to the type of `source_input` blocks that can be configured for a task given the task's `condition` block and `services` field. This is because a task cannot have multiple configurations defining the same type of monitored variable:
+- A task cannot be configured with a `condition` and `source_input` block of the same type. For example: configuring `condition "consul-kv"` and `source_input "consul-kv"` will error because both configure the `consul_kv` variable.
+- A task cannot be configured with two or more `source_input` blocks with the same type. For example: configuring `source_input "catalog-services"` twice within a task will error because those are multiple configurations for the `catalog_services` variable.
+- A task monitoring services can only configure one of `condition "services"`, `source_input "services"`, `services` list because all configure the `services` variable.
```

Comment:
```suggestion
- A task monitoring service can only have one of the following options applied; `condition "services"`, `source_input "services"`, or `services` list. This restriction is because all three options configure the `services` variable.
```

Is there a better term to use over "option"?

Reply:
Thanks for the suggestion. What are your thoughts on "A task that monitors services"? I want to keep "service" plural since it can monitor many services.

Better term for "option" 🤔. Maybe "...all three configurations define the `services` variable"?

Reply:
Thumbs up to "A task that monitors services". 
```suggestion
- A task that monitors services can only contain one of the following configurations: 
* `condition "services"`
* `source_input "services"` 
* `services <list>` 
All of the listed configurations affect the `services` variable and including more than one type will return an error.
```
Adding a nested list might not work, so I think you'd have to see how it renders. I don't know if `services <list>` is correct, either.